test/test_draft.py

```diff
@@ -39,24 +46,21 @@ def test_get_draft_returns_draft(test_draft):
     assert isinstance(test_draft.rounds[0].picks[0].person, Person)
 
 
+def test_get_latest_draftee_returns_pick(test_latest_draft_pick):
+    assert isinstance(test_latest_draft_pick, Pick)
+    assert test_latest_draft_pick.person.draftYear == 2017
+    assert isinstance(test_latest_draft_pick.person, Person)
```

Comment:
I don't know that this line and the next three are really necessary. The first two are good though.